docs/community/tutorials-contest/2022/avalanche-theGraph-index/README.md

```diff
@@ -0,0 +1,381 @@
+
+* Select a symbol for the native Subnet token
+  ![symbol](./images/4.png "token symbol")
+
+* Set fees: Select the `low disk use / low throughput` option
+  ![fees](./images/5.png "fees")
+
+* Airdrop: default to airdrop 1 million tokens to provided address
+  ![airdrop](./images/6.png "airdrop")
+
+* Add a custom precompile to modify the EVM: For this section, we will not be using a pre-compile script
+  ![precompile](./images/7.png "precompile")
+
+
+The wizard won't customize every aspect of the Subnet-EVM genesis for you, we will be doing this in the subsequent sections.
+
+![complete](./images/8.png "complete")
+
+To view the list of all created Subnets, just execute the following command
+
+```zsh
+avalanche subnet list
+```
+![list](./images/9.png "list")
+
+
+3. Deploying the Subnet Locally.
+
+To deploy the newly created Subnet locally, run the following command
+
+```zsh
+avalanche subnet deploy <SubnetName>
+```
+
+![deploy](./images/10.png "deploy")
+
+When a Subnet is run locally, it starts a multi-node (5 node) Avalanche Network in the background.
+
+![deploy_f](./images/11.png "deploy_f")
+
+To test the functionality of the just created Subnet, go ahead and add the configuration details to [Metamask](https://metamask.io/).
+You can create a new metamask account by importing the private key `0x56289e99c94b6912bfc12adc093c9b51124f0dc54ac7a766b2bc5ccf558d8027` and start experiencing with this account.
+
+I have a [Github Tutorial](https://github.com/FibrinLab/Avalanche-Local-Environment-Setup) that explains how to setup your local development environmet including `Metamask`. 
+
+Lastly don't forget to stop the running local network
+```zsh
+avalanche network stop <snapshotName>
+```
+
+![deploy_f](./images/12.png "deploy_f")
+
+
+
+### Deploying Smart Contracts with Remix
+
+1. Setting up Remix
+
+We are going to be making use of an online IDE in compiling and deploying our smart contract code, a tool called [Remix](https://remix.ethereum.org/).
+
+Navigate to the [Remix](https://remix.ethereum.org/) platform and import the test contract we will be making use of.
+
+![github](./images/22.png "github")
+
+```zsh
+https://github.com/FibrinLab/example-subgraph/blob/master/contracts/Gravity.sol
+```
+
+This repo contains the official sample subgraph for the [gravatar](https://en.gravatar.com/) registry.
+
+1. Deployment Steps
+
+Compile and deploy the `Gravity` smart contract using the Local Subnet you just created. To do this select the `deploy` tab and choose `injected web3` from the dropdwown. `Please note that Remix automatically detects the appropriate compiler version and makes use of it to compile your contract.
+
+![remix1](./images/23.png "remix1")
+
+![remix2](./images/24.png "remix2")
+
+`Always make sure to confirm the Environmet Chain ID is the same as that of your selected metamask account`.
```

Comment:
Could you use [Docusaurus notes formatting](https://docusaurus.io/docs/markdown-features/admonitions) here? And in any other notes, warnings and similar?